engine/runtime-parser/src/test/java/org/enso/compiler/core/EnsoParserTest.java

```diff
@@ -1246,6 +1246,40 @@ public void testSkip() throws Exception {
     equivalenceTest("a = x", "a = SKIP FREEZE x.f y");
   }
 
+  @Test
+  public void testBlockSyntax() throws Exception {
+    equivalenceTest("""
+    nums v fm ff n = v . map fm . filter ff . take n
+    """, """
+    nums v fm ff n = v
+        . map fm
+        . filter ff
+        . take n
+    """);
+  }
+
+  @Test
+  public void testBlockSyntaxOperators() throws Exception {
+    equivalenceTest("""
```

Comment:
Thanks for [your help](https://github.com/enso-org/enso/pull/8415#issuecomment-1831264370) @somebody1234 - here is the same test.

However I have a question: what shall happen with operator priorities? If I use `+` on the first operator line and `*` on the second operator line, shall the order be kept or should `*` take precedence? CCing @kazcw

Reply:
spec excerpt, relevant parts highlighted:

![image](https://github.com/enso-org/enso/assets/4046547/301b98e5-d4ac-4c41-b6e8-eb32bf7c1666)

Reply:
Thank you. The expression is _"built line-by-line"_ as the [next test](https://github.com/enso-org/enso/pull/8415/files#r1411551881) shows.